lib/analysis/references.dart

```diff
@@ -49,7 +49,9 @@ class FindReferencesHelper implements Disposable {
         } else {
           bool isMethod = result.element.parameters != null;
           String name = "${result.element.name}${isMethod ? '()' : ''}";
-          FindReferencesView.showView(new ReferencesSearch('References', name, searchId: result.id),
+          Future<List<SearchResult>> resultsFuture = analysisServer.waitForSearchResults(result.id);
```

Comment:
waitForSearchResults --> searchResults?
The "waitFor" implies synchronous to me and this returns with a future.